assets/uswds/js/uswds-init.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-var */
+(function uswdsInit() {
+  "use strict";
+
+  var loadingClass = "usa-js-loading";
+  var fallback;
+
+  document.documentElement.classList.add(loadingClass);
+  function revertClass() {
+    document.documentElement.classList.remove(loadingClass);
+  }
+
+  fallback = setTimeout(revertClass, 8000);
+
+  function verifyLoaded() {
+    if (window.uswdsPresent) {
+      clearTimeout(fallback);
+      revertClass();
+      window.removeEventListener("load", verifyLoaded, true);
+    }
+  }
+
+  window.addEventListener("load", verifyLoaded, true);
+})();
```

Comment:
We should add this to the head include to prevent flash of unstyled content

See additional info and example in:
https://designsystem.digital.gov/documentation/developers/#using-uswds-css-and-javascript-in-your-project:~:text=Initializer%3A%20This,dist/js.

Reply:
Excellent catch, I've moved this to the header to load first.